src/components/Fields/ContractorPOC/ContractorPOC.tsx

```diff
@@ -0,0 +1,56 @@
+import { NewPCOL } from "src/api/PCOL/types";
+import { useFormContext } from "react-hook-form";
+import { TextFieldIcon } from "@fluentui/react-icons-mdl2";
+import { useParams } from "react-router";
+import BACCombobox from "src/components/BaseFormFields/BACCombobox";
+import { useEffect } from "react";
+import { useContracts } from "src/api/Contracts/Contracts";
+import { useContractorPOCs } from "src/api/Contracts/ContractorPOCs";
+
+type ContractorPOCs = { children: string; value: string }[];
+
+export const ContractorPOC = () => {
+  const { program } = useParams();
+  const contracts = useContracts(program ?? "");
+  const contractorPOCs = useContractorPOCs(program ?? "");
```

Comment:
This is returning empty if there is a Contract POC without valid Zod data -- aka missing a phone, email, and or Contractor